tests/SnapshotTest.php

```diff
@@ -7,14 +7,23 @@
 use Illuminate\Support\Facades\Artisan;
 use Spatie\Snapshots\MatchesSnapshots;
 
-class SnapshotTest extends TestCase
+final class SnapshotTest extends TestCase
```

Comment:
Any specific reason for using `final`on this? I wonder what the benefits are.

Reply:
I always use final on tests as there's absolutely no need to extend a test class.

Reply:
Ok.